docs/source/software/basic_flight_controller.rst

```diff
@@ -1,3 +1,86 @@
 =======================
 Basic Flight Controller
 =======================
+
+The flight controller is responsible for passing high level position information to the px4. The Controller is written in C++ and designed to be extended for multiple use cases. To develop your own controller see the controller development page.
+
+This page will cover how to use the basic pose controller:
+
+* `Usage`_
+* `Troubleshooting`_
+* `ROS Parameters`_
+
+Usage
+=====
+
+You'll need to launch two sets of launch files **on the quad** via ``ssh``.
+
+ROS Bridge
+^^^^^^^^^^^^^
+::
+
+    roslaunch mslquad quad_vrpn.launch
+
+This will create the connection between the roscore, motion capture, and the px4. You should verify that the quad ID in::
+
+    quad_vrpn.launch
+
+is the same as the one you have in Motive *e.g.,* for quad3 the ID="3".
+
+Controller
+^^^^^^^^^^^^^
+::
+
+    roslaunch mslquad pose_controller.launch
+
+This will launch the ``pose_controller`` node which allows you to set a goal point of the quad. You can directly publish to ``quad#/command/pose``. You can also run::
+
+    roslaunch mslquad goal_set.launch
+
+to use a python script to set the goal position. Make sure you change the namespace before using.
+
+Troubleshooting
+===============
+.. TODO: make this section a whole other page?
+
+Things will likely break, this is here so you don't break as well. The first you should do it restart everything on the quad and make sure everything is plugged in
+
+Very many things can go wrong after a crash or coffee break, this section is here to help you sort that out. All the examples use "quad0", change that to the quad that's not working.
```

Comment:
> Things will likely break, this is here so you don't break as well. The first you should do it restart everything on the quad and make sure everything is plugged in

> Very many things can go wrong after a crash or coffee break, this section is here to help you sort that out. All the examples use "quad0", change that to the quad that's not working.

Loving the personal touch to this, but the first sentence in each of these is comparable, maybe condense it into one sentence or just pick one of them.

I would also emphasize the first point about restarting everything and making sure it is plugged in. Maybe make this a "note" https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html.

Also "plugged in" is missing a period.

Reply:
> Loving the personal touch to this, but the first sentence in each of these is comparable, maybe condense it into one sentence or just pick one of them.
good point

> I would also emphasize the first point about restarting everything and making sure it is plugged in. Maybe make this a "note" https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html.
cool, I didn't know this was a feature

> Also "plugged in" is missing a period.
good copy

Reply:
I agree with @exoticDFT on condensing down a bit.